interpreter/nodev8/v8.go

```diff
@@ -1460,33 +1459,45 @@ func decodePosition(table []byte, delta uint64) sourcePosition {
 	}
 }
 
-// mapPositionToLine maps a file position (byte offset) to a line number. This is
-// done against a table containing a offsets where each line ends.
-func mapPositionToLine(lineEnds []uint32, pos int32) libpf.SourceLineno {
+// mapPositionToLine maps a file position (byte offset) to a line number and column
+// number. This is done against a table containing offsets where each line ends.
+func mapPositionToLine(lineEnds []uint32, pos int32) (libpf.SourceLineno, libpf.SourceColumn) {
 	if len(lineEnds) == 0 || pos < 0 {
-		return 0
+		return 0, 0
 	}
 	// Use binary search to locate the line number
 	index := sort.Search(len(lineEnds), func(ndx int) bool {
 		return lineEnds[ndx] >= uint32(pos)
 	})
-	return libpf.SourceLineno(index + 1)
+
+	// Calculate column: position - start of line
+	// The start of line is the end of previous line + 1 (or 0 for first line)
+	var lineStart uint32
+	if index > 0 {
+		lineStart = lineEnds[index-1] + 1
+	} else {
+		lineStart = 0
+	}
```

Comment:
The else block is unnecessary as uninitialized variables are 0 by default in go.